delta/service/src/main/scala/ch/epfl/bluebrain/nexus/delta/service/utils/EventLogUtils.scala

```diff
@@ -27,4 +29,41 @@ object EventLogUtils {
           )
         ) >> UIO.pure(None)
     }
+
+  /**
+    * Provide extension methods for an EventLog of Envelope[Event]
+    *
+    * @param eventLog the eventLog
+    */
```

Comment:
since `fetchStateAt` has been added, shouldn't it be used in other implementations as well? `OrganizationsImpl`, `RealmsImpl`, `AclsImpl`, ...?

Reply:
Yep, I just wanted to be validated here before replicate it in the other modules.